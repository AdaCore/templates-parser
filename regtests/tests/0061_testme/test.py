from test_support import *

run('testme', ["testme60.tmplt"])
