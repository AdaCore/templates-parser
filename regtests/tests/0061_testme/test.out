
one
two
three

three
two
one

A4.1
A4.2
A4.3
A3.1
A3.2
A3.3
A2.1
A2.2
A2.3
A1.1
A1.2
A1.3

A4.3
A4.2
A4.1
A3.3
A3.2
A3.1
A2.3
A2.2
A2.1
A1.3
A1.2
A1.1

A1.3
A1.2
A1.1
A2.3
A2.2
A2.1
A3.3
A3.2
A3.1
A4.3
A4.2
A4.1

